.circleci/config.yml

```diff
@@ -336,6 +336,17 @@ workflows:
           <<: *release-branches
       - github-release: *release-tags
 
+  deploy:
```

Comment:
I like that this is split in two deploy workflows now. Maybe we should do the same for other repositories

Reply:
Yeah I think it's better organized this way, specially considering CircleCI doesn't seem to consider jobs after the tag job part of the same workflow from what I've seen.